upcloud/service/kubernetes_test.go

```diff
@@ -17,12 +17,10 @@ import (
 
 const exampleClusterResponse = `
 {
+	"control_plane_ip_filter":null,
```

Comment:
I might have left this out on purpose, as API side has `omitempty` defined for this(?), but not should not make a difference.